hawc/apps/lit/api.py

```diff
@@ -268,27 +268,6 @@ def update_reference_metadata_from_hero(self, request, pk):
         )
         return Response(status=status.HTTP_204_NO_CONTENT)
 
-    @action(
-        detail=True,
-        methods=("post",),
-        url_path="reference-search",
-        permission_classes=(AssessmentReadPermissions,),
-    )
-    def reference_search(self, request, pk):
```

Comment:
I ended up removing this API and the things around it; the only place we were using it was the reference search, and now that's a pure django view. We can potentially add back in the lit sprint as we need to, but for now, since it's not being used, I removed it.